README.md

```diff
@@ -109,4 +109,11 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/DVELP/
 
 ## License
 
-The gem is available as open source under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+Seoable is copyright of DVELP Ltd. It is free software and available to distribute under the terms of the [MIT License](http://opensource.org/licenses/MIT).
+
+## DVELP
+![alt text](https://raw.githubusercontent.com/DVELP/cookbook/assets/assets/d-for-dvelp.png  "DVELP")
```

Comment:
We could put entire logo here (all letters)?

"alt text" should be "DVELP logo" or similar.